hub/public-endpoints.mdx

```diff
@@ -0,0 +1,249 @@
+---
+title: "Public endpoints"
+sidebarTitle: "Public endpoints"
+description: "Test and deploy production-ready AI models using public endpoints."
+tag: "BETA"
+---
+
+<Note>
+**Public endpoints are currently in beta.** We're actively expanding our model selection and working to improve the user experience. [Join our Discord](https://discord.gg/runpod) if you'd like to provide feedback.
+</Note>
+
+RunPod public endpoints provide instant access to state-of-the-art AI models through simple API calls, with an API playground available through the [RunPod Hub](/hub/overview).
+
+## Available models
+
+Our initial launch includes optimized text-to-image generation models:
+
+| Model | Description | Endpoint URL |
+|-------|-------------|----------|
+| **Flux Dev** | High-quality image generation with excellent prompt adherence | `https://api.runpod.ai/v2/black-forest-labs-flux-1-dev/` |
+| **Flux Schnell** | Fast image generation optimized for speed | `https://api.runpod.ai/v2/black-forest-labs-flux-1-schnell/` |
+
+## Public endpoint playground
+
+The public endpoint playground provides a streamlined way to discover and experiment with AI models.
+
+The playground offers:
+
+- **Interactive parameter adjustment**: Modify prompts, dimensions, and model settings in real-time.
+- **Instant preview**: Generate images directly in the browser.
+- **Cost estimation**: See estimated costs before running generation.
+- **API code generation**: Create working code examples for your applications.
+
+### Access the playground
+
+1. Navigate to the [RunPod Hub](https://www.runpod.io/console/hub) in the console.
+2. Find the **Public endpoints** section.
+3. Use the dropdown menu to browse available models and select one that fits your needs.
```

Comment:
This needs to be changed. 

2. Select **Public endpoints** section. 
3. Browse available models and select one that fits your needs.